### PR TITLE
I63 create sign up requests page

### LINKED ISSUE #63: Create sign up requests page
## Basic Information
- Buttons for approve & reject
- Ability to edit the role
- Information on first & last name, email and role
- Ability to search and filter by all fields

### PR DESCRIPTION
**Pull request by Ch3ngL0rd**
## Change Summary
Created the sign-up page with basic javascript dummy functions. Added CSS to try and match the Figma file.

### Change Form

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information

# Related issue

- Resolve #63

### CODE REVIEW

File: pages/signup_requests.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            </option>
+            <option
+              value="Adjudicator Coordinator"
+              :selected="
+                person.role === 'Adjudicator Coordinator' ? true : false
+              "
+            >
+              Adjudicator Coordinator
+            </option>
+          </select>
```
Comment (ChantelleKerr) [inline pages/signup_requests.vue:66]: rather than doing selected you can do this `<select v-model="person.role">`

File: data/pretendpeople.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "first": "John",
+    "last": "Brown",
+    "email": "johnbrown@gmail.com",
+    "role": "Adjudicator Coordinator"
+  },
+  {
+    "first": "Michael",
+    "last": "Johnson",
+    "email": "michaeljohnson@gmail.com",
+    "role": "Administrator"
```
Comment (ChantelleKerr) [inline data/pretendpeople.json:18]: Public users wont be able to register as an admin - this is something that that an existing admin will handle. The options are Adjudicator, Adjudicator Coordinator, Team Coordinator.
:)

**Comment by Ch3ngL0rd** [comment]
All changes done! 😁

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Just a small change and a small warning in the console that a found

**Review by ChantelleKerr — APPROVED** [review]
Good job! Approved :D

**Pull request merged**